### PR TITLE
Bump coingecko test

### PR DESCRIPTION
**Pull request by alejoberardino**

### CODE REVIEW

File: packages/sources/coingecko-test/src/crypto-utils.ts [hunk 1]
```diff
@@ -30,7 +27,7 @@ export const cryptoInputParams: InputParameters = {
     description: 'The symbol of the currency to convert to',
     required: true,
   },
-}
+} as const
```
Comment (boxhock) [inline packages/sources/coingecko-test/src/crypto-utils.ts:30]: What does this do? Isn't it already `const`?

File: packages/sources/coingecko-test/src/crypto-utils.ts [hunk 2]
```diff
@@ -1,18 +1,15 @@
-import { AxiosRequestConfig, AxiosResponse } from 'axios'
```
Comment (boxhock) [inline packages/sources/coingecko-test/src/crypto-utils.ts:1]: I already did the work for this in #2222. Wanna get that in first, or is that redundant with this PR?

Comment (alejoberardino) [reply packages/sources/coingecko-test/src/crypto-utils.ts:1]: I think we can get this one in first which is newer, and then I can rebase the other one and check if there are pending changes

File: packages/sources/coingecko-test/src/crypto-utils.ts (hunk 1 above)
Comment (boxhock) [reply packages/sources/coingecko-test/src/crypto-utils.ts:30]: Got it, thanks!

**Review by boxhock — APPROVED** [review]

**Pull request merged**